src/sanitize-svg.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @fileOverview Sanitize the content of an SVG aggressively, to make it as safe
+ * as possible
+ */
+const fixupSvgString = require('./fixup-svg-string');
+const DOMPurify = require('dompurify');
+
+const sanitizeSvg = {};
+
+DOMPurify.addHook(
+    'beforeSanitizeAttributes',
+    currentNode => {
+
+        if (currentNode && currentNode.href && currentNode.href.baseVal &&
+            currentNode.href.baseVal.replace(/\s/g, '').slice(0, 5) !== 'data:'){
+
+            if (currentNode.attributes.getNamedItem('xlink:href')){
```

Comment:
Does this cover [this case](https://github.com/LLK/scratch-svg-renderer/pull/334/files#diff-3ad1eec0aaceeab21d48a1d174e58e4d727f61288bf4ed3dc602445739e743cbR146) from the old tests, where the `xlink` namespace is renamed?

Reply:
Not in this part of the code, but that should be handled by DOMPurify. We are testing that case in the tests we added, using the files embedded-cat.svg and embedded-cat.sanitized.svg .